bower.json

```diff
@@ -1,18 +1,15 @@
 {
   "name": "ember-resolver",
-  "dependencies": {
-    "ember": "2.0.0",
-    "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.5",
-    "ember-cli-test-loader": "ember-cli-test-loader#0.1.3",
-    "ember-load-initializers": "ember-cli/ember-load-initializers#0.1.5",
-    "ember-qunit": "0.4.10",
-    "ember-qunit-notifications": "0.0.7",
+  "devDependencies": {
+    "ember": "components/ember#release",
```

Comment:
@stefanpenner I took the ember-data approach for the version, should I go back to `2.2.0`? and should I squash this into one commit?